caktus_primer/static/stylus/variables.styl

```diff
@@ -11,4 +11,4 @@ $line-height-small = $font-size-small * 1.5
 $line-height-very-small = $font-size-very-small * 1.5
 $line-height-heading = $font-size-base * 1.25
 
-$content-tags = 'h1','h2','h3','h4','h5','pre','blockquote','p'
\ No newline at end of file
+$content-tags = 'h1','h2','h3','h4','h5','pre','blockquote','p','li'
```

Comment:
Adding `li` effects the the styles of the navigation.  I know this is a controversial topic, but given how ubiquitous `li`s are used in the real world for navigation, etc, I think we should remove this here.